README.rst

```diff
@@ -110,7 +110,7 @@ Quick Example
     bot.run('token')
 
 
-You can find more examples in the examples directory.
+You can find more examples in the `examples directory <https://github.com/nextcord/nextcord/blob/master/examples/>`_.
```

Comment:
Hmmm... on pypi I would want it to redirect to stable... but on github I would want it to redirect it to master... not quite sure how I would be able to do something like that

Reply:
Technically, it could do a a replace in the `setup.py`, but it would be cleaner to leave it unchanged

Reply:
For example

```py
readme = f.read().replace("/master/", f"/{version}/")
```

Reply:
Yeah that's not ideal... Hmmm

Reply:
Maybe make stable the default?

Reply:
maybe not cuz it affect pool request

Reply:
*pull

Reply:
or make a pypi readme file and a normal readme file ig